### DISCUSSION TITLE
Do pleasuredome DATs work?

### DISCUSSION THREAD

**Discussion by mattieb** [emmercm/igir#446] [Q&A] [answered]
I have figured out how to use igir with No-Intro DATs and it's perfect. 🤩 

It's unclear to me if I can use it with e.g. pleasuredome MAME DATs, or other MAME DATs. Since they list "machine" elements instead of "game", I'm guessing that may be unimplemented?

Although igir appears to find all the ROMs inside the MAME ZIPs, all ZIPs end up listed as "UNMATCHED" in the report.

**Comment by emmercm** [comment, marked as answer]
I'm glad `igir` has worked for at least one of your use cases @mattieb !

For MAME, I'm admittedly not the most familiar, but this third-party documentation leads me to believe you're right about the XML format: https://github.com/SabreTools/SabreTools/wiki/DatFile-Formats#mame-list-xml-format

It's interesting that I don't see anything about "machine" in the official documentation: https://web.archive.org/web/20230619034434/http://www.logiqx.com/DatFAQs/ (wayback machine because the site is always offline...)

Fingers crossed, it shouldn't be complicated for me to make this fix!

**Reply by emmercm** [reply]
https://github.com/emmercm/igir/issues/456
